build.fsx

```diff
@@ -75,6 +75,7 @@ module Test =
     let private runTests filter framework =
         !! "**/*.fsproj"
         ++ "**/*.csproj"
+        -- "Wiring/*.fsproj"
```

Comment:
Your project is at `Wiring/TickSpec.Xunit/TickSpec.Xunit.fsproj`. That is not matched using the above pattern. What you want is either `Wiring/**/*.fsproj` or `Wiring/*/*.fsproj` - that depends whether we want to allow nested folders in the `Wiring` folder. I would say that we do.

Reply:
Thanks, @mchaloupka, this is fixed now.